syntax_test_wolfram_language.wl

```diff
@@ -148,34 +148,33 @@
 (* FUNCTIONS *)
 
   f[x_, y_] := 2x
-(*^ entity.name.function*)
+(*^ entity.name.function *)
 (* ^ meta.arguments.wolfram punctuation.section.brackets.begin.wolfram *)
 (*  ^^ meta.arguments.wolfram meta.pattern.blank.wolfram variable.parameter.wolfram *)
 (*    ^^ meta.arguments.wolfram punctuation.separator.sequence.wolfram *)
-(*      ^ variable.parameter*)
+(*      ^ variable.parameter *)
 (*        ^ meta.arguments.wolfram punctuation.section.brackets.end.wolfram *)
-(*          ^^ keyword.operator*)
+(*          ^^ keyword.operator *)
 
   f[x_, OptionsPattern[]] := 2x
-(*^ entity.name.function*)
-(*  ^ variable.parameter*)
-(*      ^^^^^^^^^^^^^^ variable.function*)
-(*                        ^^ keyword.operator*)
+(*^ entity.name.function *)
+(* ^ variable.parameter *)
```

Comment:
Only the first test was wrong. Now the parameter is not matching (same in the ones below).